README.md

```diff
@@ -30,6 +30,7 @@ $ npm run dev
 - Create a new file: `config/local.js`. Override the required config values from `config/development.js` and `config/default.js` into `config/local.js`.
 - Register the application for [GitHub OAuth](https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps) to get the `clientId` and `clientSecret`. Add the callback URL as `http://<HOSTNAME>:<PORT>/auth/github/callback`
 - Create an application on [FireStore](https://firebase.google.com/docs/firestore) and [generate a service file](https://cloud.google.com/iam/docs/creating-managing-service-account-keys). Add the service file with the name `firestore-private-key.json` in the project root. 
+- Check your local environment is properly setup by running command `npm run test-config`.
```

Comment:
How about:

Check if your local environment is set up by running the following command:
`npm run test-config`.

Reply:
What about: 

Make sure you local development setup is working properly by running the following command:
```npm  run test-config```

Reply:
test-config command is to check whether files needed are properly inserted in the project folder or not
So I think 'setup is working properly' should be used here.

Reply:
Sure. Whatever you prefer. I just gave my suggestion.

Also, one correction in your statement and it is good to go.

Make sure **your** local development setup is working properly by running the following command:
`npm run test-config`

Reply:
@swarajpure Please approve and merge this PR.